lib/rollbar/plugins/active_job.rb

```diff
@@ -1,35 +1,44 @@
-module Rollbar
-  # Report any uncaught errors in a job to Rollbar and reraise
-  module ActiveJob
-    def self.included(base)
-      base.send :rescue_from, Exception do |exception|
-        args = if self.class.respond_to?(:log_arguments?) && !self.class.log_arguments?
-                 arguments.map(&Rollbar::Scrubbers.method(:scrub_value))
-               else
-                 arguments
-               end
+Rollbar.plugins.define('active_job') do
```

Comment:
👋 - this change produces:

```
NameError: uninitialized constant Rollbar::ActiveJob (NameError)

  include Rollbar::ActiveJob
                 ^^^^^^^^^^^
```
which means that https://docs.rollbar.com/docs/activejob-integration is no longer true. Is there a change required to the documentation?

https://docs.rollbar.com/docs/gem-plugins is unclear on how plugins are loaded, so we were unable to experiment with a solution.